docs/content/guides/6.multistore/2.tooling-and-concepts/4.deployment/6.deployment.md

```diff
@@ -7,14 +7,147 @@ navigation:
 
 # Deployment - How it works?
 
+Ever wondered how Alokai transforms your codebase into a live, running application? This guide takes you behind the scenes of the deployment process, showing you exactly how your code goes from development to production.
+
+## Deployment Process Overview
+
+Ready to deploy your store? It all starts with a single command:
+
+```bash
+yarn store deploy
+```
+
+While this command is typically executed in your [continuous delivery pipeline](/guides/multistore/tooling-and-concepts/deployment/ci-cd), you can also run the deployment locally on your machine.
+
+::tip Getting Started with Deployment
+Learn how to configure your stores and run your deployment locally in the [Deployment - Configuration](/guides/multistore/tooling-and-concepts/deployment/configuration) guide. For more details about the `store deploy` command, see the [CLI reference](/guides/multistore/tooling-and-concepts/cli-reference).
+::
+
+The command performs a deployment of chosen stores. The deployment process consists of six main stages that transform your source code into running applications:
+
+1. Store Composition
+2. Build Process
+3. Docker Image Preparation
+4. Docker Image Building
+5. Image Registry Push
+6. Deployment Trigger
+
+Let's explore each stage in detail.
+
+### 1. Store Composition
+
+The journey begins with store composition. The CLI uses the [file-based inheritance](/guides/multistore/tooling-and-concepts/file-based-inheritance) rules to compose the stores by:
+- Collecting files from base applications
+- Applying overrides from parent stores, based on the file-based inheritance
+- Adding store-specific customizations
+
+Everything comes together in the temporary `.out/<store-id>` directory (which is git-ignored), ready for the next stage.
+
+::info Independent Files in `.out` Directory
+Unlike in the `apps` directory where files are shared between stores through inheritance, each store in the `.out` directory has its own independent copy of all files. Let's take a look at the following example:
+
+```bash
+apps/
+├── storefront-unified-nextjs/          # Base shared code
+    ├── tailwind.config.ts        
+    ├── ...
+└── stores/
+    ├── fashion-brand/        
+    │   ├── storefront-unified-nextjs/ 
+    │   │   ├── components/ 
+    │   │   │   ├── header.tsx 
+    │   ├── stores/
+    │   │   ├── us/
+    │   │   ├── eu/
+```
+
+The CLI copies files to their respective store directories, creating a fully independent version of each store:
```

Comment:
It might not be clear what _"independent version"_ means. I'd write explicitly that they are separate next/nuxt projects / source code.

Reply:
good point https://github.com/vuestorefront/vue-storefront/pull/7400/commits/6f91dde8c709b7285c6dd7a2c12d71bacbd72041